src/backend/api/Fusion.Resources.Api/Fusion.Resources.Api.csproj

```diff
@@ -19,14 +19,14 @@
 
 	<ItemGroup>
 		<PackageReference Include="Bogus" Version="34.0.2" />
-		<PackageReference Include="Fusion.Integration" Version="6.0.0" />
+		<PackageReference Include="Fusion.Integration" Version="6.0.3" />
 		<PackageReference Include="Fusion.Events.Server" Version="3.0.0" />
 		<!-- No longer need this reference, but keeping for future ref. -->
 		<!--<PackageReference Include="Fusion.Integration.Profile.Abstractions" Version="5.1.0" />-->
```

Comment:
Not yours to change, but not sure why we need this for future ref... (ref. the one I asked to remove)